pkg/appStore/AppStoreDeploymentService.go

```diff
@@ -1163,15 +1165,15 @@ func (impl InstalledAppServiceImpl) AppStoreDeployOperationGIT(installAppVersion
 	valid, err := chartutil.IsChartDir(chartPath)
 	if err != nil || !valid {
 		impl.logger.Errorw("invalid base chart", "dir", chartPath, "err", err)
-		return nil, nil, err
+		return installAppVersionRequest, nil, err
```

Comment:
why did we need to change from nil to installAppVersionRequest. As per golang, we need to first check fir error and not rely on values in case error is not nil